Lib/sqlite3/test/dbapi.py

```diff
@@ -933,6 +934,72 @@ def test_on_conflict_replace(self):
         self.assertEqual(self.cu.fetchall(), [('Very different data!', 'foo')])
 
 
+class MultiprocessTests(unittest.TestCase):
+    def setUp(self):
+        with sqlite.connect(TESTFN) as cx:
+            cx.execute("create table t(t)")
+
+    def tearDown(self):
+        unlink(TESTFN)
+
+    def test_rollback_if_db_is_locked(self):
+        # bpo-27334: ctx manager does not rollback if commit fails
+        TIMEOUT = 0.01
```

Comment:
Why this small TIMEOUT value? (Notice the test suite uses some customizable timeout values that yuou may want to reuse here)

Reply:
> Why this small TIMEOUT value?

The default connection timeout is very long. The test depends on the SQLite timing out when it tries to commit. Using a small timeout value makes it possible to execute the test very fast, thus not slowing down the CI.

> (Notice the test suite uses some customizable timeout values that yuou may want to reuse here)

Oh, I did not know; I'll check it out. Thanks :)